plugins/python-build/bin/python-build

```diff
@@ -1538,13 +1538,14 @@ use_homebrew_yaml() {
 
 use_macports_yaml() {
   can_use_macports || return 1
-  local prefix="$(port -q location libyaml 2>/dev/null || true)"
+  local port_location="$(command -v port)"
+  local prefix="${port_location%/bin/port}"
   if [ -n "$prefix" ]; then
-    local libdir="$prefix/opt/local"
-    if [ -d "$libdir" ]; then
+    local installation="$(port -q list libyaml)"
```

Comment:
This line is for checking if the library is installed.
`installation` is probably not the best name. Please suggest a better one.